eliot/_traceback.py

```diff
@@ -13,16 +13,18 @@
 from ._validation import MessageType, Field
 from ._errors import _error_extraction
 
+
+def class_fqpn(typ):
+    """Convert a class to it's fully-qualified name."""
+    return "%s.%s" % (typ.__module__, typ.__name__)
```

Comment:
Can't you just use `__qualname__`?
It's available since Python 3.3.

Reply:
Python 2.7 is still supported.

Reply:
Not anymore! But when that code was written, it was supported.